colossalai/shardformer/policies/llama.py

```diff
@@ -91,6 +96,7 @@ def module_policy(self) -> Dict[Union[str, nn.Module], ModulePolicyDescription]:
                 target_key=attn_cls,
             )
 
+        # if self.pipeline_stage_manager is not None:
```

Comment:
why

Reply:
> why
Removed in 
https://github.com/hpcaitech/ColossalAI/pull/6083/commits/fafe049b83bad3a6aa6e3a31c68b38ac63167b53